index.html

```diff
@@ -1,23 +1,29 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
     <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
     <header id="header">
-            <nav class="navbar">
-                <a href=""><h2>My Logo</h2></a>
-                <img src="images/Hamburger_button.svg" alt="menu button">
-            </nav>
+        <nav class="navbar">
+            <a href="">
+                <h2>My Logo</h2>
+            </a>
+            <img src="images/Hamburger_button.svg" alt="menu button">
+        </nav>
     </header>
-    
+
     <section id="headline">
         <h1>I’m Mariane Glad to see you!</h1>
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```

Comment:
- [ ] I think it's better to avoid lines of code longer than 100 characters following the best practices of coding. For more information [click here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
This was for the first part of the project, and it got approved. This is not part of this project requirement. Thank you.